docs/impact_health.rst

```diff
@@ -0,0 +1,62 @@
+#################
+Impact and Health
```

Comment:
I know I said this to you already, but this text is great!

Reply:
Thanks!